.github/workflows/ci.yml

```diff
@@ -21,13 +20,27 @@ jobs:
           MYSQL_USER: drupal
           MYSQL_PASSWORD: drupal
           MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
         options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=10s --health-retries=10
+      drupal:
+        image: phenaproxima/acquia_cms:headless
```

Comment:
How should we keep this up to date in future? Should we give at least some other folks permissions to push to this for future use?

Reply:
Honestly, it should be on Acquia CMS to provide container images of their software. They can use my work here as a model. Once I've finished writing the build scripts, it should be possible to keep it pretty easily up to date. I'm open to granting push access to one or two Acquia CMS maintainers, but ideally they'd just provide their own "official" images.

Reply:
I think we must give someone else access there just for the long term maintainability of this package. It's not a massive deal even if we lose access to the image but ideally we would not have to redo that work in future 😅

Reply:
I don't object, but to be clear, this image is not meant to be long-term maintainable. We need to coordinate with the Acquia CMS team to come up with a plan here. (I don't think that should block merging this PR, but we should alert them of this need.)

Reply:
I have now access to the image so that at least addresses my immediate concerns. I'd like to still see further work to make maintaining the image easier but I think what we have there now works for now.